CREATE-PLAY.md

```diff
@@ -29,18 +35,18 @@ Welcome developers! We are as excited as you are to know that you are going to c
 - Fill the information and submit.
   Parameter details
   
```

Comment:
@Susmita-Dey Why is this table portion below changed?

Reply:
Might be due to formatting the code